pyproject.toml

```diff
@@ -23,8 +23,12 @@ seaborn = ">=0.11.1,<0.13.0"
 scikit-learn = "^1.1.1"
 sympy = "^1.10.1"
 tinkerforge = {version = "^2.1.25", optional=true}
-torch = "2.0.0"
 tqdm = "^4.64.0"
+torch = [
```

Comment:
for my own education, will dependabot go in and update each of these going forward?

Reply:
I'm actually not sure on this one, I'll have to look into the dependabot docs. However I've not come to the correct syntax yet for this solution as all the builds have failed... 😅 Though it will look something like this with different dependency calls based on system platform.

Reply:
gotcha!